Tips 'n tricks/Swift-Objective C interoperability and best practices.md

```diff
@@ -134,6 +133,83 @@ If you need to use Swift code in `.h` file, you should use something called Forw
 
 ![1](/img/iOS-objc-swift-interoperability.png)
 
+### Swift enums and Objective-C
+
+#### Header Files - `.h`
+
+>IMPORTANT: Swift enums can't be used in Objective-C header files.
+
+But if your project must have Swift enums in header files, there are two possible solutions:
+
+ 1. Write Swift enum in Objective-C and use newly written enum in Swift files. By doing this, you will have codebase that is easy to maintain.
+
+ 2. Have enum written both in Objective-C and Swift.
+
+ Although having duplicated code is inherently a bad thing, and it will be more challenging to maintain the codebase, using this solution your code will become more flexible and here are the perks that come with it:
```

Comment:
```suggestion
 Although having duplicated code is inherently a bad thing, and it will be more challenging to maintain, using this solution your code will become more flexible and here are the perks that come with it:
```